chump/__init__.py

```diff
@@ -582,7 +582,7 @@ def __setattr__(self, name, value):
 		
 		elif name in ('message', 'title', 'url', 'url_title', 'device', 'callback', 'sound'):
 			if value is not None:
-				if not isinstance(value, basestring):
```

Comment:
Honestly, I don't know why I got an error here when using it as from source installed library. This is what I found as solution from google.